howdoi/howdoi.py

```diff
@@ -37,14 +27,18 @@
 from requests.exceptions import ConnectionError
 from requests.exceptions import SSLError
 
-# Handle unicode between Python 2 and 3
-# http://stackoverflow.com/a/6633040/305414
```

Comment:
Ok. I thought it was implicit while mentioning the `u` function. I will update the pull request correspondingly.